build-package

```diff
@@ -11,10 +11,10 @@ OUTDIR="$STARTDIR/deb"
 rm -rf "$DESTDIR"
 
 # Install the package itself
-npm install -g --prefix "$DESTDIR/usr" thelounge@2.5.0
+npm install --unsafe-perm -g --prefix "$DESTDIR/usr" thelounge@2.7.0
```

Comment:
Could we infer version from `debian/control` somehow?

Reply:
Probably, but that belongs to a further pr :)

Reply:
You already do something similar with `DEBFILE` in tests it seems